### ISSUE TITLE
Error each time running EnvyUpdate

### ISSUE THREAD

**Issue by Fooughhy** [fyr77/EnvyUpdate#28]
I'm having an issue with EnvyUpdate which I think makes it not work at all at the moment (I wondered why I wasn't prompted to install new drivers when they were released, but I knew EnvyUpdate was scheduled to run on startup).

In EventHistory there are two "Errors" that appear each time I try to start EnvyUpdate. This is regardless if I start it normally, as admin, or as a scheduled task.

The first error is the following (unfortunately in Swedish):
```
Tillämpningsprogram: EnvyUpdate.exe
Framework-version: v4.0.30319
Beskrivning: Processen avslutades på grund av ett ohanterat undantag.
Undantagsinformation: System.IO.DirectoryNotFoundException
   vid System.IO.__Error.WinIOError(Int32, System.String)
   vid System.IO.FileStream.Init(System.String, System.IO.FileMode, System.IO.FileAccess, Int32, Boolean, System.IO.FileShare, Int32, System.IO.FileOptions, SECURITY_ATTRIBUTES, System.String, Boolean, Boolean, Boolean)
   vid System.IO.FileStream..ctor(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, Int32, System.IO.FileOptions, System.String, Boolean)
   vid System.Net.FileWebStream..ctor(System.Net.FileWebRequest, System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, Int32, Boolean)
   vid System.Net.FileWebResponse..ctor(System.Net.FileWebRequest, System.Uri, System.IO.FileAccess, Boolean)

Undantagsinformation: System.Net.WebException
   vid System.Net.FileWebResponse..ctor(System.Net.FileWebRequest, System.Uri, System.IO.FileAccess, Boolean)
   vid System.Net.FileWebRequest.GetResponseCallback(System.Object)

Undantagsinformation: System.Net.WebException
   vid System.Net.WebClient.DownloadDataInternal(System.Uri, System.Net.WebRequest ByRef)
   vid System.Net.WebClient.DownloadString(System.Uri)
   vid System.Net.WebClient.DownloadString(System.String)
   vid EnvyUpdate.MainWindow+<Load>d__9.MoveNext()
   vid System.Runtime.CompilerServices.AsyncMethodBuilderCore+<>c.<ThrowAsync>b__6_0(System.Object)
   vid System.Windows.Threading.ExceptionWrapper.InternalRealCall(System.Delegate, System.Object, Int32)
   vid System.Windows.Threading.ExceptionWrapper.TryCatchWhen(System.Object, System.Delegate, System.Object, Int32, System.Delegate)
   vid System.Windows.Threading.DispatcherOperation.InvokeImpl()
   vid System.Windows.Threading.DispatcherOperation.InvokeInSecurityContext(System.Object)
   vid MS.Internal.CulturePreservingExecutionContext.CallbackWrapper(System.Object)
   vid System.Threading.ExecutionContext.RunInternal(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   vid System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   vid System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object)
   vid MS.Internal.CulturePreservingExecutionContext.Run(MS.Internal.CulturePreservingExecutionContext, System.Threading.ContextCallback, System.Object)
   vid System.Windows.Threading.DispatcherOperation.Invoke()
   vid System.Windows.Threading.Dispatcher.ProcessQueue()
   vid System.Windows.Threading.Dispatcher.WndProcHook(IntPtr, Int32, IntPtr, IntPtr, Boolean ByRef)
   vid MS.Win32.HwndWrapper.WndProc(IntPtr, Int32, IntPtr, IntPtr, Boolean ByRef)
   vid MS.Win32.HwndSubclass.DispatcherCallbackOperation(System.Object)
   vid System.Windows.Threading.ExceptionWrapper.InternalRealCall(System.Delegate, System.Object, Int32)
   vid System.Windows.Threading.ExceptionWrapper.TryCatchWhen(System.Object, System.Delegate, System.Object, Int32, System.Delegate)
   vid System.Windows.Threading.Dispatcher.LegacyInvokeImpl(System.Windows.Threading.DispatcherPriority, System.TimeSpan, System.Delegate, System.Object, Int32)
   vid MS.Win32.HwndSubclass.SubclassWndProc(IntPtr, Int32, IntPtr, IntPtr)
   vid MS.Win32.UnsafeNativeMethods.DispatchMessage(System.Windows.Interop.MSG ByRef)
   vid System.Windows.Threading.Dispatcher.PushFrameImpl(System.Windows.Threading.DispatcherFrame)
   vid System.Windows.Threading.Dispatcher.PushFrame(System.Windows.Threading.DispatcherFrame)
   vid System.Windows.Application.RunDispatcher(System.Object)
   vid System.Windows.Application.RunInternal(System.Windows.Window)
   vid System.Windows.Application.Run(System.Windows.Window)
   vid EnvyUpdate.App.Main()
```
And the second error is:
```
Felet uppstod i programmet med namn: EnvyUpdate.exe, version 2.13.0.0, tidsstämpel 0xe1b77e98
, felet uppstod i modulen med namn: KERNELBASE.dll, version 10.0.19041.2364, tidsstämpel 0x1e5c8604
Undantagskod: 0xe0434352
Felförskjutning: 0x0012e292
Process-ID: 0x532c
Programmets starttid: 0x01d91b56d62f6893
Sökväg till program: C:\Users\fough\scoop\apps\envyupdate\current\EnvyUpdate.exe
Sökväg till modul: C:\WINDOWS\System32\KERNELBASE.dll
Rapport-ID: fc2f6219-5230-46bb-9aa2-f6227c680e22
Fullständigt namn på felaktigt paket: 
Program-ID relativt till felaktigt paket: 
```

I am running Windows 10 Pro (19045.2364) and EnvyUpdate 2.13.0.0 (as can be seen in the error). And I am also using scoop to update the tool (and have disabled auto-update). Though I can see now that scoop doesn't have the absolut latest version yet (should be 2.15, right)?

**Comment by fyr77** [comment]
Hello! That was a bug I fixed in 2.15.
I removed my scoop repository because of lack of time and motivation to keep it up, so EnvyUpdate does currently not update via scoop.
I would ask you to remove it from scoop and manually download the newest version from GitHub here.

Another option is to use winget. Another person added EnvyUpdate to that package manager. Keep in mind that is not maintained by me, but that seems to work as well.

**Comment by Fooughhy** [comment]
Ah sorry about that. I only saw issue #23 which was related to crashes, so I thought that was the one which was fixed with the new release. But I see now that that issue was created in January, so I should have known! Thanks, it seems to work now!
